src/outputs/tau.py

```diff
@@ -33,7 +33,7 @@ def output_tau(
         ultfoc_mapper (pd.DataFrame): The ULTFOC mapper DataFrame.
         cora_mapper (pd.DataFrame): used for adding cora "form_status" column
         postcode_itl_mapper (pd.DataFrame): maps the postcode to region code
-        pg_alpha_num (pd.DataFrame): mapper of numeric PG to alpha PG
```

Comment:
I think the number of arguments for this function is wrong (sorry would make the suggestion but github won't let me!).

Here's what I'm getting:
![image](https://github.com/ONSdigital/research-and-development/assets/73113361/4707e7c6-1ce3-4cb1-808c-a15ae5cf5b89)